thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -360,6 +352,10 @@ class CDistributorBase : public CSimpleInterface, implements IHashDistributor, i
             {
                 return bucket.getClear();
             }
+            bool queryMarkSenderFinished()
+            {
+                return 0 != atomic_cas(&senderFinished, 1, 0);
```

Comment:
I have to apologise - atomic_cas returns a boolean so my comment was wrong - needs changing back